recipes/sequana/meta.yaml

```diff
@@ -1,21 +1,17 @@
-{% set version = "0.1.21" %}
-{% set name = "sequana" %}
-{% set md5 = "e19d1b27200976b899f91e0f85ef0d3f" %}
-
 package:
-  name: {{ name }}
-  version: {{ version }}
+  name: sequana
+  version: "0.2.0"
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Any reason why you reverted the nicer template syntax from the previous version?